requirements.txt

```diff
@@ -1,2 +1,2 @@
 django~=1.11.18
-git+https://github.com/unt-libraries/edtf-validate.git@master#egg=edtf-validate
+git+https://github.com/unt-libraries/edtf-validate.git@py3#egg=edtf-validate
```

Comment:
This should stay master, I am waiting for an approval on edtf-validate, and then it will be merged to master as it still works in py2 as well.